lib/client/identityfile/identity.go

```diff
@@ -614,10 +614,14 @@ func KeyFromIdentityFile(identityPath, proxyHost, clusterName string) (*client.K
 // is also added to its profile store with the limited profile data available
 // in the identity file.
 //
-// Since identity files do not save a proxy address, proxyAddr must be provided
-// to fill in this data gap. clusterName can also be provided to aim the key at
-// a leaf cluster rather than the default root cluster.
+// Use [proxyAddr] to specify the host:port-like address of the proxy.
+// This is necessary because identity files do not store the proxy address.
+// Additionally, the [clusterName] argument can ve used to target a leaf cluster
+// rather than the default root cluster.
 func NewClientStoreFromIdentityFile(identityFile, proxyAddr, clusterName string) (*client.Store, error) {
+	if proxyAddr == "" {
+		return nil, trace.BadParameter("Missing a Proxy address when loading an Identity File.")
```

Comment:
Nit: Do not capitalize errors for better embedding (from Go style guide).
```suggestion
		return nil, trace.BadParameter("missing a proxy address when loading an identity file")
```